scripts/build-plugin.mjs

```diff
@@ -0,0 +1,108 @@
+import fs from 'node:fs/promises'
+import { statSync } from 'node:fs'
+import path from 'node:path'
+import process from 'node:process'
+
+const sourceSrc = `${process.cwd()}\\dist`
+const targetSrc = `${process.cwd()}\\browserPlugin`
```

Comment:
<img width="754" alt="image" src="https://github.com/cumtflyingstudio/atcumt/assets/79413249/e7d82ade-d9f2-470f-9c4c-c8a0a1c7d2c6">
There are cross-platform incompatibility in your code, such as splicing file paths, where node handles cross-platform processing.

Try using the `path.join(xxx, 'xxx')` API to fix this error  :D

Reply:
const str = path.join(process.cwd(), 'dist')
In windows  => it will be `C:\\Windows\\System\\xxx\\dist`
In posix  => it will be `/home/xxx/xx/dist`

More information can be seen [#windows-vs-posix](https://nodejs.dev/en/api/v19/path/#windows-vs-posix) :D